guides/source/getting_started.md

```diff
@@ -7,1150 +7,1548 @@ This guide covers getting up and running with Ruby on Rails.
-articles. So before we proceed to the "C", "U", and "D" sections, let's replace
-the two `get` routes in `config/routes.rb` with `resources`:
+To define a route in Rails, let's go back to your code editor and add the following route to `config/routes.rb`
 
 ```ruby
 Rails.application.routes.draw do
-  root "articles#index"
-
-  resources :articles
+  get "/products", to: "products#index"
 end
 ```
 
-We can inspect what routes are mapped by running the `bin/rails routes` command:
+This route tells Rails to look for GET requests to the `/products` path. In this example, we specified `"products#index"` for where to route the request.
 
-```bash
-$ bin/rails routes
-      Prefix Verb   URI Pattern                  Controller#Action
-        root GET    /                            articles#index
-    articles GET    /articles(.:format)          articles#index
- new_article GET    /articles/new(.:format)      articles#new
-     article GET    /articles/:id(.:format)      articles#show
-             POST   /articles(.:format)          articles#create
-edit_article GET    /articles/:id/edit(.:format) articles#edit
-             PATCH  /articles/:id(.:format)      articles#update
-             PUT    /articles/:id(.:format)      articles#update
-             DELETE /articles/:id(.:format)      articles#destroy
-```
-
-The `resources` method also sets up URL and path helper methods that we can use
-to keep our code from depending on a specific route configuration. The values
-in the "Prefix" column above plus a suffix of `_url` or `_path` form the names
-of these helpers. For example, the `article_path` helper returns
-`"/articles/#{article.id}"` when given an article. We can use it to tidy up our
-links in `app/views/articles/index.html.erb`:
-
-```html+erb
-<h1>Articles</h1>
-
-<ul>
-  <% @articles.each do |article| %>
-    <li>
-      <a href="<%= article_path(article) %>">
-        <%= article.title %>
-      </a>
-    </li>
-  <% end %>
-</ul>
-```
-
-However, we will take this one step further by using the [`link_to`](
-https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to)
-helper. The `link_to` helper renders a link with its first argument as the
-link's text and its second argument as the link's destination. If we pass a
-model object as the second argument, `link_to` will call the appropriate path
-helper to convert the object to a path. For example, if we pass an article,
-`link_to` will call `article_path`. So `app/views/articles/index.html.erb`
-becomes:
-
-```html+erb
-<h1>Articles</h1>
-
-<ul>
-  <% @articles.each do |article| %>
-    <li>
-      <%= link_to article.title, article %>
-    </li>
-  <% end %>
-</ul>
-```
+When Rails sees a request that matches, it will send the request to the `ProductsController` and the `index` action inside of that controller. This is how Rails will the request and return a response to the browser.
```

Comment:
When Rails sees a request that matches, it will send the request to the `ProductsController` and the `index` action inside of that controller. This is how Rails will map the request and returns a response to the browser.